### PR TITLE
fluent-ffmpeg update

### PR DESCRIPTION
**Pull request by jgainfort**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/fluent-ffmpeg/node-fluent-ffmpeg#reading-video-metadata
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

ffprobe can exist both on the ffmpeg namespace and as an instance member, see url above.

Also with ffprobe(options: string[], ...) the options param is a string not a string array, see url above.

### CODE REVIEW

**Comment by jgainfort** [comment]
@yuit 
Those two authors never gave any feedback on my last PR for this library typings. https://github.com/DefinitelyTyped/DefinitelyTyped/pull/16471

And that PR was in queue for 21 days.

**Comment by yuit** [comment]
@jgainfort  thanks for head up!... one quick comment could you add  the examples in the attached link in the tests?

File: types/fluent-ffmpeg/index.d.ts [hunk 1]
```diff
@@ -279,7 +287,7 @@ declare namespace Ffmpeg {
         /* tslint:disable:unified-signatures */
         ffprobe(callback: (err: any, data: FfprobeData) => void): (err: any, data: FfprobeData) => void;
         ffprobe(index: number, callback: (err: any, data: FfprobeData) => void): (err: any, data: FfprobeData) => void;
-        ffprobe(options: string[], callback: (err: any, data: FfprobeData) => void): (err: any, data: FfprobeData) => void;
```
Comment (mhegazy) [inline types/fluent-ffmpeg/index.d.ts]: looks like string is supported.. see https://github.com/fluent-ffmpeg/node-fluent-ffmpeg/blob/b6b6ef35d3c09d6dc71a23ceb1fc25bccb7acb1f/lib/ffprobe.js#L119

Comment (jgainfort) [inline types/fluent-ffmpeg/index.d.ts]: I agree. I will put that back.

**Comment by jgainfort** [comment]
@mhegazy 

So after further use of the library I cannot get it to support a string array. Even though I agree with you that at first look at the library and this line https://github.com/fluent-ffmpeg/node-fluent-ffmpeg/blob/master/lib/ffprobe.js#L119

Implementing it like

```
import ffmpeg = require('fluent-ffmpeg')

ffmpeg.ffrpobe(['my/input/path'], (err, data) => {
  ...
})
```

Always returns this error

> Error: No input specified\n
>   at FfmpegCommand.proto.ffprobe (/usr/src/app/node_modules/fluent-ffmpeg/lib/ffprobe.js:128:31)\n    at Function.FfmpegCommand.ffprobe (/usr/src/app/node_modules/fluent-ffmpe
> g/lib/fluent-ffmpeg.js:221:20)\n    at Promise (/usr/src/app/dist/adaptors/encoders/ffmpeg.js:138:20

So can I put this PR on hold until I have a chance to talk with maintainers of fluent-ffmpeg and get this sorted out?

**Comment by minestarks** [comment]
@jgainfort I'm holding off on merging this upon your request. If it's not sorted within a couple of weeks we'll close it out for inactivity, then you can open a new PR.

**Comment by sandersn** [comment]
Thanks for your contribution. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the master branch.

**Pull request closed without merging**